src/bindings/js/node/lib/addon.ts

```diff
@@ -205,6 +205,13 @@ interface Core {
     deviceName: string,
     properties: { [key: string]: string | number | boolean },
   ): void;
+  /**
+   * It queries the model passed for the requested property.
+   * @param model The passed model to query the property.
+   * @param deviceName The name of a device.
+   * @param properties An object with the property name - property value pairs.
```

Comment:
```suggestion
   * @param properties An object with the property name - property value pairs. An object with the key-value pairs
   * (property name, property value)
```
In other places it looks like this

Reply:
Done